doc/Tribler REST API.md

```diff
@@ -27,6 +33,26 @@ The API has been built using [Twisted Web](http://twistedmatrix.com/trac/wiki/Tw
 | ---- | --------------- |
 | GET /variables | Returns runtime-defined variables used by the current Tribler session |
 
+## `GET /channels/subscribed`
+
+Returns all the channels the user is subscribed to.
```

Comment:
Ah it should probably say `returns all the channels you are subscribed to` to make it more clear. I will change it.